okapi-operation/src/axum_integration/mod.rs

```diff
@@ -35,8 +39,21 @@ use crate::*;
         )
     )
 )]
-pub async fn serve_openapi_spec(spec: State<OpenApi>) -> Json<OpenApi> {
-    Json(spec.0)
+pub async fn serve_openapi_spec(spec: State<OpenApi>, headers: HeaderMap) -> Response {
+    match headers.get("Accept").and_then(|h| h.to_str().ok()) {
+        Some("yaml") => AxumYaml(spec.0).into_response(),
```

Comment:
I thought it might be a good idea to put yaml integration under feature, someone might want to have as little dependencies as possible.

Reply:
Fixed